app/controllers/finders_controller.rb

```diff
@@ -7,7 +7,10 @@ def show
     @results = ResultSetPresenter.new(finder, view_context)
 
     respond_to do |format|
-      format.html
+      format.html do
+        @content_item = Services.content_store.content_item(finder_base_path).to_hash
```

Comment:
We're already doing a request to get the content item (somewhere in `FinderApi`). We should reuse that so we don't do two requests.